### PR TITLE
Add TaxationForm model

### PR DESCRIPTION
**Pull request by VDmitryO**

### CODE REVIEW

File: db/migrate/20180808060717_add_deadline_to_taxation_form.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddDeadlineToTaxationForm < ActiveRecord::Migration[5.2]
+  def change
+    add_column :taxation_forms, :deadline, :integer
```
[the remark this answers is not in the document]
Comment (VDmitryO) [reply db/migrate/20180808060717_add_deadline_to_taxation_form.rb]: we don't have a fixed period, we only know the date to which we need to pay

[the remark this answers is not in the document]
Comment (allomov) [reply db/migrate/20180808060717_add_deadline_to_taxation_form.rb]: shouldn't that go to `Tax` model then? by period I mean how often you need to pay this tax.

[the remark this answers is not in the document]
Comment (VDmitryO) [reply db/migrate/20180808060717_add_deadline_to_taxation_form.rb]: I think creating this period when generating taxes is easier

[the remark this answers is not in the document]
Comment (allomov) [reply db/migrate/20180808060717_add_deadline_to_taxation_form.rb]: then you don't need `deadline` field here.

[the remark this answers is not in the document]
Comment (VDmitryO) [reply db/migrate/20180808060717_add_deadline_to_taxation_form.rb]: `deadline`  depends on `TaxationForm`. Why can't I leave him here?

[the remark this answers is not in the document]
Comment (allomov) [reply db/migrate/20180808060717_add_deadline_to_taxation_form.rb]: - `period_type` ("year", "quarter", "month", "weak")
- `declaration_period_duration_in_days` (for instance 20 will mean that you need to pay till 20th day of a month)

File: db/seeds.rb [hunk 2]
```diff
@@ -6,7 +6,17 @@
...
 
+User.create(email: 'test@mail.ru', password: '123456', password_confirmation: '123456')
+
 OrganizationForm.create(name: 'Физическое лицо (самозанятость)')
 OrganizationForm.create(name: 'Индивидуальный Предприниматель')
 OrganizationForm.create(name: 'Частное предприятие')
 OrganizationForm.create(name: 'OOO')
+
+TaxationForm.create(name: 'Единый налог', organization_form: OrganizationForm.first)
+TaxationForm.create(name: 'УСН', organization_form: OrganizationForm.second)
```
Comment (allomov) [inline db/seeds.rb]: добавь данные о периоде

**Pull request merged**